layers/meta-balena-5x-owa/recipes-bsp/u-boot/files/0001-Fix-reboot-issue-upgrade-available-init-value-is-0.patch

```diff
@@ -0,0 +1,25 @@
+From fe48b57945cc6d7dbe5014be8cd264da6f53fb37 Mon Sep 17 00:00:00 2001
+From: Alvaro Guzman <alvaro.guzman@owasys.com>
+Date: Mon, 24 Apr 2023 12:06:20 +0200
+Subject: [PATCH] Fixing reboot issue upgrade available init value is 0
+
+---
+ include/configs/imx8mp_owa5x.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/include/configs/imx8mp_owa5x.h b/include/configs/imx8mp_owa5x.h
+index d10bf19259..e49eca6047 100644
+--- a/include/configs/imx8mp_owa5x.h
++++ b/include/configs/imx8mp_owa5x.h
+@@ -189,7 +189,7 @@
+ 	"fdt_addr=0x88000000\0"			\
+ 	"fdt_high=0xffffffffffffffff\0" \
+ 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0" \
+-	"upgrade_available=1\0" \
```

Comment:
why is this set in the BSP?

Reply:
Because we do use a custom mechanism to update our devices.

Reply:
hello @floion I am closing this PR and splitting it in several and smaller ones. The patch that fixes the reboot problem is now found in this [PR](https://github.com/balena-os/balena-owa5x/pull/105). Can you take a look at it please?